xclim/indices/_threshold.py

```diff
@@ -268,6 +309,43 @@ def freshet_start(
     return group.apply(rl.first_run_ufunc, window=window, index="dayofyear")
 
 
+@declare_units("C days", tas="[temperature]", thresh="[temperature]")
+def freezing_degree_days(
```

Comment:
I think this is the same as `xclim.indices.heating_degree_days` (but with different thresholds)!